packages/app/src/elements/item-view.ts

```diff
@@ -654,6 +671,47 @@ export class ItemView extends Routing(StateMixin(LitElement)) {
                             </div>
                         </div>
 
+                        <div class="history">
+                            <h2
+                                class="subtle horizontally-double-margined bottom-margined animated section-header"
+                                style="margin-left: 1.2em;"
+                            >
+                                <pl-icon icon="history" class="inline small light"></pl-icon>
+                                ${$l("History")}
+                            </h2>
+
+                            <pl-list class="border-top block" ?hidden=${!history.length}>
+                                ${history.map((historyEntry) => {
+                                    const historyEntryUpdater =
+                                        org && org.getMember({ accountId: historyEntry.updatedBy });
+                                    // TODO: Remove this
+                                    console.log(historyEntry);
```

Comment:
This showed me that on save, the `.history[]` entries have everything (including `.name` or `.tags` when I changed those), but right after sync (< 2 seconds, locally), those disappear. I still have `.fields`, for some reason. Ideas @MaKleSoft ?